#vector of the pose of all the drones
droneInfo[] societyDrone

